### PR TITLE
Update roslyn to use newer VSIX installer (master)

### PR DESCRIPTION
**Pull request by jmarolf**
This will fix installer issues for dogfood customers

### CODE REVIEW

File: src/Test/Perf/Utilities/VsPerfTest.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public abstract class VsPerfTest : PerfTest
         private readonly string[] _scenarios;
         private static readonly string _nugetPackagesPath = System.Environment.GetEnvironmentVariable("NUGET_PACKAGES") ??
             Path.Combine(System.Environment.GetEnvironmentVariable("UserProfile"), ".nuget", "packages");
-        private static readonly string _installerPath = Path.Combine(_nugetPackagesPath, "roslyntools.microsoft.vsixexpinstaller", "0.4.0-beta", "tools", "vsixexpinstaller.exe");
```
Comment (tmat) [inline src/Test/Perf/Utilities/VsPerfTest.cs]: Can we flow this path at runtime (as a parameter) or from the build environment (e.g. [like so](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/MSBuildTaskTests/Microsoft.Build.Tasks.CodeAnalysis.UnitTests.csproj#L43-L56))?

Comment (jmarolf) [reply src/Test/Perf/Utilities/VsPerfTest.cs]: I don't know any of infra for VsPerfTest.cs .  If @agocke is ok with this change I'll make it in a subsequent PR.

Comment (jasonmalinowski) [reply src/Test/Perf/Utilities/VsPerfTest.cs]: Isn't this code dead?

Comment (jmarolf) [reply src/Test/Perf/Utilities/VsPerfTest.cs]: ¯\_(ツ)_/¯

**Comment by agocke** [comment]
(Aside from the fact that it looks like your package doesn't exist).

**Comment by jmarolf** [comment]
Looks like a flaky test:

```ini
    Microsoft.CodeAnalysis.Editor.UnitTests.Rename.RenameTagProducerTests.CSharp_FixupSpanDuringResolvableConflict_ComplexificationReordersReferenceSpans [FAIL]
```

**Comment by jmarolf** [comment]
@dotnet-bot test windows_release_unit32_prtest please

**Pull request merged**